src/sentry/static/sentry/app/utils/tokenizeSearch.tsx

```diff
@@ -344,20 +351,12 @@ function isSpace(s: string) {
  */
 function formatTag(tag: string) {
   const idx = tag.indexOf(':');
-  const key = stripSurroundingQuotes(tag.slice(0, idx));
-  const value = stripSurroundingQuotes(tag.slice(idx + 1));
+  const key = tag.slice(0, idx).replace(/^"+|(?<!\\)"+$/g, '');
+  const value = tag.slice(idx + 1).replace(/^"+|(?<!\\)"+$/g, '');
```

Comment:
From slack, negative lookbehind isn't supported by safari 🙄 will have to look at another solution
https://caniuse.com/js-regexp-lookbehind